protocol/syncer.go

```diff
@@ -295,7 +277,6 @@ func (s *Syncer) updatePeerStatus(peerID peer.ID, status *Status) {
 		status.Number,
 		"latest block hash",
 		status.Hash, "difficulty",
```

Comment:
Should we reove "difficulty" from debug log?

Reply:
Brought it back in the latest commit that preserves backwards compatibility:

[138ea36](https://github.com/0xPolygon/polygon-edge/pull/524/commits/138ea36c43d91280270d57ca27bf7bda1fddb24a)

Reply:
Thanks, good catch!